pkg/libvmi/storage.go

```diff
@@ -82,9 +82,14 @@ func WithEmptyDisk(diskName string, bus v1.DiskBus, capacity resource.Quantity)
 
 // WithCDRom specifies a CDRom drive backed by a PVC to be used.
 func WithCDRom(cdRomName string, bus v1.DiskBus, claimName string) Option {
+	return WithCDRomAndVolume(bus, newPersistentVolumeClaimVolume(cdRomName, claimName))
+}
+
+// WithCDRom specifies a CDRom drive backed by given volume.
```

Comment:
Still:
```suggestion
// WithCDRomAndVolume specifies a CDRom drive backed by given volume.
```
Still a nit though, still feel free to unhold ;)

Reply:
My apology.